### PR TITLE
Added a simplified version of LoRaWAN USe-Case

### PR DESCRIPTION
**Pull request by sandoche2k**
Simplified LoRaWAN Use Case version

### CODE REVIEW

**Comment by sandoche2k** [comment]
@oej I have simplified the text by removing application-specific parameters.

**Comment by mcr** [comment]
Too much text for the architecture document, seems like the start of a DANCEr document.

**Comment by sandoche2k** [comment]
@mcr further simplified the LoRaWAN use-case

File: draft-ietf-dance-architecture.md [hunk 1]
```diff
@@ -190,6 +190,14 @@ Using DANE for device identity can allow parties other than the implementer to o
 A hardware manufacturer can provide a pre-established identity, with the certificate or public key already published in DNS.
 This makes PKI-based identity more approachable for small organizations which currently lack the resources to operate an organizational CA.
 
+### LoRaWAN
+
+**We should ask S. if he wants to contribute to this section**
```
Comment (mcr) [inline draft-ietf-dance-architecture.md:195]:
```suggestion

```

**Review by mcr — APPROVED** [review]

File: draft-ietf-dance-architecture.md [hunk 2]
```diff
@@ -190,6 +190,14 @@ Using DANE for device identity can allow parties other than the implementer to o
...
+
+For the end-device onboarding in LoRaWAN, the "network server" and the "join server" [RFC 8376] needs to establish mutual TLS authentication in order to exchange configuration parameters. Certificate Authority based mutual TLS authentication doesn't work in LoRaWAN due to the non availability of the CA trust store in the LoRaWAN network stack. Self-signed certificate based mutual-TLS authentication method is the alternative solution.
```
Comment (mcr) [inline draft-ietf-dance-architecture.md:197]:
```suggestion
For the end-device onboarding in LoRaWAN, the "network server" and the "join server" {{?RFC8376}} needs to establish mutual TLS authentication in order to exchange configuration parameters. Certificate Authority based mutual TLS authentication doesn't work in LoRaWAN due to the non availability of the CA trust store in the LoRaWAN network stack. Self-signed certificate based mutual-TLS authentication method is the alternative solution.
```

**Review by oej — APPROVED** [review]
This looks good.

**Review by mcr — APPROVED** [review]

**Pull request merged**